3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -134,6 +134,7 @@ function add_member() {
     info "ERROR: couldn't join to replica set!"
     info "CAUSE: failed after waiting for becoming a member. Command output was:"
     echo "${rs_status_out}"
+    echo "==> End of the error output <=="
```

Comment:
This is yet another format for log messages, is this something you're intending to have in the image or just debugging to be removed before merge?